.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+	env: {
+		browser: true,
+		es2020: true
+	},
+	extends: "eslint:recommended",
+	parserOptions: {
+		sourceType: "module"
+	},
+	rules: {
+		indent: ["error", "tab", {
+			SwitchCase: 1
+		}],
+		"no-unused-vars": "warn",
```

Comment:
I don't know if you like to keep them for documentation, or not. If they should be removed, let me know and I'll add another commit to the PR.